src/services/item/plugins/importExport/service.ts

```diff
@@ -200,17 +201,18 @@ export class ImportExportService {
       reply;
       item: Item;
```

Comment:
Is it possible to type it here as a `DiscriminatedItem`? This would allow the extras to be typed correctly below without the need to cast them specifically

Reply:
I will do the change, but it does not fit well with the Item repository. For example get an item return type is `Item` which does not fit with `DiscriminatedItem`. Changing this would result in changes everywhere in the project. 

So I've opted for casting to `Item` instead, which is indeed better.